### PR TITLE
Feature/cordoplug 473

### PR DESCRIPTION
**Pull request by Archifer**

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -52,5 +49,4 @@
     </onegini>
     <plugin name="cordova-plugin-dialogs" spec="~1.3.0" />
     <plugin name="cordova-plugin-pin-dialog" spec="~0.1.3" />
-    <plugin name="cordova-plugin-onegini" spec="^9.1.0" />
```
Comment (MichielVrins) [inline config.xml:55]: why are we removing the plugin here?

File: src/views/Image-upload.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    methods: {
+      uploadImage: function (event) {
+        let imageFile = event.target.files[0];
+        this.baseImageUpload(imageFile);
+      },
+
+      baseImageUpload: async function (imageFile) {
+        // set the image
+        const formData = new FormData();
+        formData.append("attachments", imageFile, "filename~き");
```
Comment (MichielVrins) [inline src/views/Image-upload.vue:29]: comment hier waarom er rare characters in zitten.

File: config.xml (hunk 1 above)
Comment (Archifer) [reply config.xml:55]: this is depricated starting from cordova 9 (we are at version 11 now)

File: src/views/Image-upload.vue (hunk 2 above)
Comment (Archifer) [reply src/views/Image-upload.vue:29]: updated

**Review by MichielVrins — APPROVED** [review]

**Pull request merged**